sources/code/main/windows/main.ts

```diff
@@ -68,6 +72,43 @@ export default function createMainWindow(flags:MainWindowFlags): BrowserWindow {
     },
     ...(process.platform !== "win32" ? {icon: appInfo.icons.app} : {}),
   });
+  
+  // Custom Bar
+  if (useCustomBar){
+    const custombarView = new BrowserView({
+      webPreferences: {
+        nodeIntegration: false,
+        preload: resolve(app.getAppPath(), "app/code/renderer/preload/custombar.js")
+      }
+    });
+    win.setBrowserView(custombarView);
+    custombarView.setAutoResize({ width: true })
+    custombarView.webContents.loadFile(resolve(app.getAppPath(), "sources/assets/web/html/custombar.html"));
+    custombarView.setBounds({ width: win.getBounds().width, height: 20, x: 0, y: 0 });
+    custombarView.webContents.on('ipc-message', (_event, channel) => {
+      switch (channel) {
+        case 'window-minimize':
+          console.debug("[Custom Bar] Minimize window...");
+          win.minimize();
+          break;
+        case 'window-maximize':
+          console.debug("[Custom Bar] Maximize window...");
+          win.isMaximized() ? win.unmaximize() : win.maximize();
+          break;
+        case 'window-close':
+          console.debug("[Custom Bar] Maximize window...");
+          win.close();
+          break;
+        case 'opensettings':
+          console.debug("[Custom Bar] Opening settings...");
+          loadSettingsWindow(win);
+          break;
+        default:
+          break;
+      }
+    });    
```

Comment:
Um... I think you should check `npm test` or `npm run lint` sometimes... :wink:

Reply:
Fixed